db/revision_cache_interface.go

```diff
@@ -33,20 +41,37 @@ var _ RevisionCache = &BypassRevisionCache{}
 // NewRevisionCache returns a RevisionCache implementation for the given config options.
 func NewRevisionCache(cacheOptions *RevisionCacheOptions, backingStore RevisionCacheBackingStore, statsCache *expvar.Map) RevisionCache {
 
+	// If cacheOptions is not passed in, use defaults
+	if cacheOptions == nil {
+		cacheOptions = &RevisionCacheOptions{Size: defaultRevisionCacheSize, ShardCount: defaultRevisionCacheShardCount}
```

Comment:
Switching this to a call to DefaultRevisionCacheOptions() would make this more future-proof.

Reply:
Oops! Yep definitely.